front/.eslintrc.js

```diff
@@ -25,6 +25,12 @@ module.exports = {
     "react-hooks/rules-of-hooks": 0,
```

Comment:
For some reason VScode flag the misused promises but the linter does not with `npm run lint`. Will investigate.